product_docs/docs/transporter/ba_preview/rel_notes/index.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "EDB Transporter Release notes"
+navTitle: "Release notes"
+description: Learn about new features and functions.
+navigation:
+- rel_notes_2.0.0_preview
+---
+
+The EDB Transporter documentation describes the latest version of EDB
+Transporter 2, including minor releases and patches. The release notes
+provide information on what was new in each release. For new functionality
+introduced in a minor or patch release, the content also indicates the release
+that introduced the feature.
+
+| Release Date | Data Migration                           |
+|--------------|------------------------------------------|
+| 2023 Jun 9   | [2.0.0_preview](rel_notes_2.0.0_preview) |
```

Comment:
What do we want to state here? 
The GA release date (in a couple of months)?
The internal preview release, in June 2024? 
Or is this date correct because there was another internal release last year?

Reply:
We've brought this stack up from prototype, to alpha, a "June 2023 Preview" release, but none of these were external facing. We can omit releases prior to the initial DMS / Transporter Reader version.

Reply:
Gotcha. I guess that means we can delete the contents of this page and replace them with a release date once we decide to go GA? Do we have an ETA for that?

Reply:
Just answered the second part of that question myself 😆  
ETA: August 15th.